### ISSUE TITLE
rClone for heterogeneous storage endpoints (suggestion/enhancement/to-think)

### ISSUE THREAD

**Issue by victorsndvg** [singularityhub/sregistry-cli#125]
Hi @vsoch ,

Last week I was taking a look to rClone (https://rclone.org/). I thought it could be interesting to be wrapped by sregistry to be able to download singularity images from non-(singlarity/docker)registry heterogeneous endpoints.

 what do you think?

**Comment by vsoch** [comment]
yeah this is a really good idea - I have seen rclone and it would be (much easier) to have sregistry wrap it and get support for all those things! It's written in go (and sregistry-cli is written in python) so I think it would only be worth pursuring given a huge revamp of the library. I don't see the push / user base that would warrant this atm, and I don't think we have the developer strength (aka just me) so I'm not going to jump on this. But agreed, great idea!
